src/arproc.py

```diff
@@ -675,7 +672,6 @@ def get_ampsec_trimmed(slf, fitsdict, det, scidx):
     # Construct and array with the rows and columns to be extracted
     w = np.ix_(xfin, yfin)
     slf._ampsec[det-1] = retarr[w]
-    return fitsdict
```

Comment:
For clarity, it might be better to keep the return statement in (but remove fitsdict)